response_play.go

```diff
@@ -0,0 +1,111 @@
+package pagerduty
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+type ResponsePlay struct {
+	APIObject
+	Name        string        `json:"name,omitempty"`
+	Description string        `json:"description,omitempty"`
```

Comment:
Because both of these are `string` and you have `omitempty`, a consumer of this package would never be able to provide an empty string to the API. This means you could never unset the name or description of a `ResponsePlay`.

According to the [API documentation](https://developer.pagerduty.com/api-reference/b3A6Mjc0ODE2Ng-create-a-response-play), neither the `name` or `description` are required. So is this the desired behavior?

Reply:
I will remove `omitempty` from description but the `name` cannot be blank. The API Documentation is wrong, it is a required field